statics/js/components/object-detail.js

```diff
@@ -91,6 +91,13 @@ Vue.component('object-detail', {
       }
     },
 
+    isnotempty: function(obj) {
```

Comment:
isNotEmpty

Reply:
JQuery implemented in a similar way by using ```in``` 
since it is available from 1.4(9 years old) I hope most of them will have that.
I am using the jquery method now.
Thanks

Reply:
thank you